src/tarantool.client/Model/TarantoolTuple.cs

```diff
@@ -30,26 +31,35 @@ public override bool Equals(object obj)
 
         public override int GetHashCode()
         {
-            var hashCode = EqualityComparer<T1>.Default.GetHashCode(Item1);
-            return hashCode;
+            unchecked
```

Comment:
Do we really need unchecked{} section here?
What about other TarantoolTuple classes (<T1,T2> and so on)?